### DISCUSSION TITLE
maximization lift coefficient for an the objective function for multiple point airfoil optimization

### DISCUSSION THREAD

**Discussion by EngJamal2023** [mdolab/MACH-Aero#135] [Q&A]
I'm hoping that the community can help me find an error in my code. I created this code to calculate optimization, and I made some modifications to it. However, I'm encountering an error and would appreciate any assistance in identifying and fixing the issue. I've included the relevant code snippet below for reference.
i use **prefoil model** to create the distribution the point around airfoil 

```
mach = [0.15, 0.044]
alt = [100, 80]
alpha = [1.5, 10.17]
mycd = [0.01, 0.024]

```
and i put some of adflow solver option for low mach number as following 
```
    "liftIndex": 2,
    "infchangecorrection": True,
    "writeTecplotSurfaceSolution": False,
    "acousticScaleFactor":0.01,
    "ANKCharTimeStepType": "VLR",
```
because i use analysi for airfoil for wind turbine i change the thickness 

`DVCon.addThicknessConstraints2D(leList, teList, 2, 100, lower=0.1, upper=1.15,scaled=True)
`
i change here the opjective function and drag constraint as following 
```
        funcs["obj"] -= funcs[ap["cl"]] / nFlowCases
        funcs["cd_con_" + ap.name] = funcs[ap["cd"]] - mycd[i]
```
also i change the scale of the objective function 

`optProb.addObj("obj", scale=1.0)`

**i hope if there is mistake fill me with correction it is so glad for your help**

**Comment by MachAeroDeveloper** [comment]
please anyone answer me please ###
